ui/packages/consul-ui/app/components/topology-metrics/card.hbs

```diff
@@ -1,5 +1,9 @@
 <a class="card"
-  href={{href-to "dc.services.show.index" @item.Name}}
+  href={{if
+   (and (env 'CONSUL_NSPACES_ENABLED') (not-eq @item.Namespace @service.Namespace))
```

Comment:
This template language is a bit difficult for me to parse - does this also cover the `else` cases properly for:
- [ ] namespaces disabled = regular service link
- [ ] namespaces enabled, service in same namespace = link to appropriately namespaced service?

Reply:
👍 Yeah it'll cover both those cases.